test/fixtures/parser/exasol/AlterConsumerGroup.sql

```diff
@@ -0,0 +1,5 @@
+ALTER CONSUMER GROUP TEST_TEAM SET
+ PRECEDENCE = '800',
+ CPU_WEIGHT = '150',
+ GROUP_TEMP_DB_RAM_LIMIT = '10G',
+ SESSION_TEMP_DB_RAM_LIMIT = '5G';
```

Comment:
Can we add newlines to end of every file?

```suggestion
 SESSION_TEMP_DB_RAM_LIMIT = '5G';
 
```suggestion
 SESSION_TEMP_DB_RAM_LIMIT = '5G';
 
```
We have a .editorconfig file to handle this automatically if your editor supports this.

Reply:
Thanks for the hint. Added EditorConfig Plugin to my VSCode.